aci/data_source_aci_fvcep.go

```diff
@@ -362,6 +369,14 @@ func dataSourceAciClientEndPointRead(d *schema.ResourceData, m interface{}) erro
 		}
 	}
 
+	if filterDn, ok := d.GetOk("filter_dn"); ok {
+		if queryString != "" {
+			queryString = fmt.Sprintf("%s,wcard(fvCEp.dn, \"%s\")", queryString, filterDn.(string))
```

Comment:
```suggestion
			queryString = fmt.Sprintf("%s,wcard(fvCEp.dn, \"%s*\")", queryString, filterDn.(string))
```

I suppose this is missing here as it is in the other else case?